app/queries/subscribers_for_immediate_email_query.rb

```diff
@@ -1,9 +1,9 @@
 class SubscribersForImmediateEmailQuery
-  def self.call
+  def self.call(filter_hash)
```

Comment:
I can see the benefit of having a generic argument here, but since this method is already highly specialiased I wonder if it's better to define it like this:

```suggestion
  def self.call(content_change_id: nil, message_id: nil)
    raise "Must specify either content_change_id or message_id" unless content_change_id || message_id
```

The advantage being that we can catch any unusual uses of this class which might not be what users expect and it helps to document this method better (the first thing I did when I saw this was look down to see what arguments are being passed to it).

Reply:
Excellent suggestion! 👍